### PR TITLE
VELO-1476: skaffold

### PR DESCRIPTION
**Pull request by crdueck**
Chart was copied from https://github.com/Wattpad/miranda/tree/master/k8s/charts/components/ship-it with some modifications for skaffold and preparation for the `syncd` & `operator` components of ship-it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,44 @@
...
+you'll need to install these tools locally:
+
+* helm: https://github.com/helm/helm
+* minikube: https://github.com/kubernetes/minikube
+* skaffold: https://github.com/GoogleContainerTools/skaffold
+
+1. Create a new minikube cluster
+
+```bash
+$ minikube create
```
Comment (rsgc-peterson-e) [inline README.md]: The command needed here is actually `minikube start`. Running this command also seemed to automatically set cluster context for `kubectl`.

File: README.md [hunk 2]
```diff
@@ -3,3 +3,44 @@
...
+```
+
+2. Update kubectl's cluster context
+
+```bash
+$ minikube update-content
```
Comment (rsgc-peterson-e) [inline README.md]: Should be `minikube update-context`

File: README.md [hunk 3]
```diff
@@ -3,3 +3,44 @@
...
+minikube
+```
+
+3. Install tiller in minikube
+
+```bash
+$ helm init
+```
+
+4. Run skaffold in development mode
```
Comment (rsgc-peterson-e) [inline README.md]: Before running `skaffold dev, the custom resource will need to be installed on the local cluster by running `kubectl apply -f deploy/v1alpha1_helmrelease_crd.yaml`.

Comment (crdueck) [reply README.md]: We'll fix this by including the CRD in the `ship-it` chart

File: deploy/ship-it/templates/_helpers.tpl [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+*/}}
+{{ define "ship-it.metadataLabels" }}
+  app: {{ template "ship-it.name" . }}
+  chart: {{ template "ship-it.chart" . }}
+  instance: {{ .Release.Name }}
+  managed-by: {{ .Release.Service }}
+{{- end -}}
+
+{{/*
+Image value hack for skaffold.
```
Comment (crdueck) [inline deploy/ship-it/templates/_helpers.tpl]: I don't really like this, but haven't found anything better yet.

**Pull request merged**